### ISSUE TITLE
Content not updated after editing view 

### ISSUE THREAD

**Issue by Meeken1998** [solid/node-solid-server#1413]
Im sorry, I tried to modify the view for node-solid-server but failed. For I used it for the first time, I tried to replace some content globally(like `<title></title>`), but the content remained the same when I restarted.

P.S. I used test mode to run this app: `bin/solid-test start`

---

Would u mind tell me how to update the UI effectively? I am a novice😂.

**Comment by jaxoncreed** [comment]
Where exactly is the document you're modifying? Without it, I can't exactly diagnose your problem. However, I might have an idea.

When you create an account, the HTML is cloned from this folder https://github.com/solid/node-solid-server/tree/master/default-views.

So, if you're modifying content in that folder, it won't automatically update. Instead, go into your `/data` folder and modify the HTML document in there.

**Comment by Meeken1998** [comment]
❤️ Thank you for your answer.  In fact, we were planning to promote SoLid in China, and  do some translation work.

---

Before I tried to modify the view, **I replaced the contents of several DOM elements globally.** I found that they did not take effect. 
Then I checked the configuration of `express`. Due to the limited time, I can only use the black box method to check.

Do u mean that the application will automatically pull the view on the server, and the local changes are invalid?
